content/fiks-plattform/tjenester/fiksprotokoll/brukerveiledning/opprette_system.md

```diff
@@ -0,0 +1,58 @@
+
+#### Steg 1: Velg Fiks Protokoll
+Finn Fiks Protokoll i listen over tjenester. Tjenester som er tatt i bruk ligger øverst. Klikk på "Fiks Protokoll"-knappen
+![fiks protokoll](/images/protokoll-brukerveiledning/2_velg_tjeneste.png "Velg tjeneste")
+
+#### Steg 2: Klikk opprett system
+Her vil det være en liste over systemer. Denne listen vil være tom dersom det ikke er blitt opprettet system.
+Klikk på "Opprett nytt system" for å opprette nytt system.
+![fiks protokoll](/images/protokoll-brukerveiledning/2_opprett_system_knapp.png "Opprett system")
+
+#### Steg 3: Wizard - Systeminfo 
+Det vil nå åpne seg en dialog med 4 steg. I første steg skal du fylle ut:
+* Systemnavn - Navnet på systemet. Dette vil være visningsnavnet til systemet i Fiks. Dette navnet vil være synlig for andre organisasjoner som komuniserer med systemet
+* Beskrivelse - En mer detaljert beskrivelse av systemet. Dette vil også være synlig for andre organisasjoner
+* E-post(er) for varsling - En liste (minimum en) av eposter som blir brukt for å kontakte ansvarlige for systemet
+
+Trykk "Neste" for å komme videre
+![fiks protokoll](/images/protokoll-brukerveiledning/2_wizard_1_utfylt.png "Wizard info")
+
+#### Steg 4: Wizard - Sett opp integrasjon 
+For å kunne sende og motta meldinger gjennom Fiks Protokoll kreves en integrasjon. Integrasjonen brukes via maskinporten, hvor et virksomhetssertifikat identifiserer eier av integrasjonen. Virksomhetssertifikatet hører til et organisasjonsnummer som må fylles ut her. I produksjon vil det variere om dette er kommunens virksomhetssertifikat eller om det er leverandørens. 
+
+NB: Det vil ikke være mulig å bruke Fiks Protokoll i test uten et virksomhetssertifikat. Det er mulig å endre organisasjonsnummer for integrasjonen senere dersom du velger feil i dette steget. [Se hvordan opprette klient hos maskinporten her](/fiks-platform/difiidportenklient)
+
+Dersom du ønsker å konfigurere systemet gjennom API kan du velge dette her. Da vil det være mulig å endre på systemet og opprette kontoer uten å bruke webgrensesnittet.
+
+Trykk "Neste" for å komme videre
+![fiks protokoll](/images/protokoll-brukerveiledning/2_wizard_2_utfylt.png "Wizard integrasjon")
+
+#### Steg 5: Wizard - Opprett system
+Da er systemet klar til å opprettes. 
+Trykk "Opprett Fiks Protokoll-system" for å opprette
+![fiks protokoll](/images/protokoll-brukerveiledning/2_wizard_3.png "Wizard opprett")
+
+#### Steg 6: Wizard - System opprettet
+Systemet er nå opprettet. 
+
+På denne siden vises nødvendig nøkkelinformasjon.
+* systemId - Denne IDen representere systemet og brukes blant annet når en konto i et annet system skal gi tilgang til dette systemet og dersom systemet skal konfigureres gjennom API
+* integrasjonId - Denne IDen brukes for maskin til maskin pålogging og kreves for tilgang til API. Denne skal inn i konfigurasjonen av Fiks IO-klienten
```

Comment:
[nit] lenke til dokumentasjon for konfigurasjon av Fiks IO-klienten hvis vi har det? 🤔

Reply:
Lenker til klientene i alle fall